### PR TITLE
Create signer_bug Issue Template

### PR DESCRIPTION
**Pull request by will-corcoran**
Adding an issue template for Signers to submit bugs and issues with the Signer

### Applicable issues

- fixes #

### Checklist

- [ ] Test coverage for new or modified code paths
- [ ] Changelog is updated
- [ ] Required documentation changes (e.g., `docs/rpc/openapi.yaml` and `rpc-endpoints.md` for v2 endpoints, `event-dispatcher.md` for new events)
- [ ] New clarity functions have corresponding PR in `clarity-benchmarking` repo
- [ ] New integration test(s) added to `bitcoin-tests.yml`

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/signer_bug [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+## Context (Environment)
+<!--- Provide some information about your environment in order to help address the issue. -->
+
+- <!--- Signer public key -->
+- <!--- Signer version -->
+- <!--- Stacks node version -->
+- <!--- Relevant log messages -->
+- <!--- Tx ID of any transaction you were trying to execute -->
+- <!--- Tx ID of `aggregation-commit-indexed` call in registered cycle (if applicable)
```
Comment (BowTiedDevOps) [inline .github/ISSUE_TEMPLATE/signer_bug:49]:
```suggestion
- <!--- Tx ID of `aggregation-commit-indexed` call in registered cycle (if applicable) -->
```

**Comment by BowTiedDevOps** [comment]
Also, the file needs to have a markdown extension:
```diff
- .github/ISSUE_TEMPLATE/signer_bug
+ .github/ISSUE_TEMPLATE/signer_bug.md
```

**Review by will-corcoran — COMMENTED** [review]
lgtm

File: .github/ISSUE_TEMPLATE/signer_bug.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+## Context (Environment)
+<!--- Provide some information about your environment in order to help address the issue. -->
+
+- <!--- Signer public key -->
+- <!--- Signer version -->
+- <!--- Stacks node version -->
+- <!--- Relevant log messages -->
+- <!--- Tx ID of any transaction you were trying to execute -->
+- <!--- Tx ID of `stack-aggregation-commit-indexed` call in registered cycle (if applicable) -->
```
Comment (BowTiedDevOps) [inline .github/ISSUE_TEMPLATE/signer_bug.md:49]: Shoot, overlooked this part - it's duplicated.
```suggestion
```

**Review by wileyj — APPROVED** [review]
lgtm

**Review by BowTiedDevOps — APPROVED** [review]
Lgtm!

**Review by obycode — APPROVED** [review]
👍

**Pull request merged**